src/ocrd_utils/logging.py

```diff
@@ -205,3 +205,8 @@ def disableLogging(silent=not config.OCRD_LOGGING_DEBUG):
     # Python default log level is WARNING
     logging.root.setLevel(logging.WARNING)
 
+# Initialize logging now to prevent any other libraries (tensorflow,
+# matplotlib ...) from adding their own root handler, which might mangle the
+# STDOUT (our logging will log to STDERR so as not to interfere with
+# --dump-json etc.)
+initLogging()
```

Comment:
No, that's not what I meant. See https://github.com/OCR-D/core/issues/1223#issuecomment-2470587136

Reply:
Alternatively, we could still do `initLogging` on the module level, though not in `ocrd_utils.logging` but rather `ocrd.decorators`, which arguably will only be imported by applications that do need log handlers.